client/components/leaders.jsx

```diff
@@ -1,31 +1,112 @@
 Leaders = React.createClass({
 
-  mixins: [ReactRouter.Navigation],
+  mixins: [ReactRouter.Navigation, ReactMeteorData],
```

Comment:
@dshvazz Cool. I'll probably need a lot of this sorting/leaderboard functionality for the admin stuff so I'll probably do a bit of abstraction here.